tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 
 - name: Compatibility check
   fail: msg="This role only works on Debian and Archlinux systems"
-  when: not ansible_os_family == "Debian|Archlinux"
+  when: ansible_os_family != "Debian" and ansible_os_family != "Archlinux"
```

Comment:
this has already been fixed. try to merge master

Reply:
oh okay i'll check